Dockerfile

```diff
@@ -168,5 +128,48 @@ RUN if [ "$TOMCAT_EXTRAS" = false ]; then \
     rm -rf $CATALINA_HOME/webapps/manager; \
   fi;
 
+# Add unlimited crypto support
+ENV JRE_HOME $JAVA_HOME/jre
+COPY ./UnlimitedJCEPolicyJDK8/*.jar $JRE_HOME/lib/security/
+
+#RUN export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib:/usr/lib/x86_64-linux-gnu:/opt/libjpeg-turbo/lib64
+
+#webxml update for environment
+RUN rm /usr/local/tomcat/webapps/geoserver/WEB-INF/web.xml
+COPY ./web.xml /usr/local/tomcat/webapps/geoserver/WEB-INF/
```

Comment:
This is generally not a good method for using the web.xml. If we change the version of Geoserver we are building then we might break Geoserver by having a different web.xml. And what changes are being introduced into the container from web.xml